.github/workflows/releasecandidate.yml

```diff
@@ -67,3 +67,17 @@ jobs:
           slack_webhook_url: ${{secrets.SLACK_WEBHOOK_URL}}
           channel: '#team-devops'
           name: 'Workflow Status'
+    
+  run-sdk-networking-tests:
+    needs: [test, staticbuild, appimage]
+    if: ${{ success() }}
+    name: Trigger Networking Tests Workflow
+    runs-on: ubuntu-latest
+    steps:
+      - name: Trigger Workflow
+        run: |
+          curl -X POST \
+          -H "Authorization: token ${{secrets.GITHUB_TOKEN}}" \
```

Comment:
I'm not sure if this is the right token needed to be able to run this curl, it needs to have multiple repo write access to work. Theoretically with the token I could run this command to see if it triggers the workflow as well

Reply:
another testing option -- make this its own file, give it a workflow_dispatch event, and run manually

(no harm in running extra sdk tests)

I have no preference though, do whatever works best for sdk team.